### PR TITLE
AArch64: enable ARM CI for kata 2.0

### PR DESCRIPTION
**Pull request by Pennyzct**
Hi guys

I've tried to enable ARM CI for kata 2.0. ;) 🎉🎊
For importing new features like rust-agent, shimv2 as default, I need to enable containerd-cri integration tests on ARM CI.
I will follow http://jenkins.katacontainers.io/job/kata-containers-2.0-tests-ubuntu-PR/ to create jenkins job for arm ci.

### CODE REVIEW

**Comment by Pennyzct** [comment]
ARM Jenkins Job for tests repo: http://jenkins.katacontainers.io/job/kata-containers-2.0-tests-ubuntu-ARM-PR/

**Comment by Pennyzct** [comment]
cc @jodh-intel @bergwolf @gnawux @chavafg @devimc @amshinde

**Comment by Pennyzct** [comment]
ARM CI is happy. ;)

**Comment by Pennyzct** [comment]
Add another jenkins job for [kata-containers/kata-containers](https://github.com/kata-containers/kata-containers): 
http://jenkins.katacontainers.io/job/kata-containers-2.0-ubuntu-ARM-PR/

File: .ci/lib.sh [hunk 1]
```diff
@@ -271,6 +271,27 @@ delete_crio_stale_resource() {
...
+	sudo systemctl stop containerd
+	# remove stale binaries
+	containerd_cri_dir="github.com/containerd/cri"
+	release_dir="${GOPATH}/src/${containerd_cri_dir}/_output/release-stage"
+	binary_dir_union=( "/usr/local/bin" "/usr/local/sbin" )
+	for binary_dir in ${binary_dir_union[@]}
+	do
+		for stale_binary in ${release_dir}/${binary_dir}/*
+		do
+			sudo rm -rf ${binary_dir}/$(basename ${stale_binary})
```
Comment (jodh-intel) [inline .ci/lib.sh:285]: If only you could call `sudo make uninstall` ;-)

File: .ci/lib.sh [hunk 2]
```diff
@@ -271,6 +271,27 @@ delete_crio_stale_resource() {
...
+		done
+	done
+	# remove cluster directory
+	sudo rm -rf /opt/containerd/
+	# remove configuration files
+	sudo rm -rf /etc/containerd/config.toml
+	sudo rm -rf /etc/crictl.yaml
```
[the remark this answers is not in the document]
Comment (Pennyzct) [reply .ci/lib.sh:292]: thanks! update asap ;)

File: .ci/lib.sh (hunk 1 above)
Comment (Pennyzct) [reply .ci/lib.sh:285]: looks to me, `make uninstall` only uninstall `/usr/local/bin/containerd`. 
On AArch64, we built from source code(`make release`), it installed the following binaries:
```
+ install bin/ctr bin/containerd bin/containerd-stress bin/containerd-shim bin/containerd-shim-runc-v1 bin/containerd-shim-runc-v2
./
./usr/
./usr/local/
./usr/local/bin/
./usr/local/bin/ctr
./usr/local/bin/containerd-shim-runc-v1
./usr/local/bin/critest
./usr/local/bin/containerd-stress
./usr/local/bin/containerd-shim-runc-v2
./usr/local/bin/containerd
./usr/local/bin/crictl
./usr/local/bin/containerd-shim
./usr/local/sbin/
./usr/local/sbin/runc

```

**Comment by bergwolf** [comment]
@Pennyzct Time to drop `(DNM)` from the PR title?

**Comment by Pennyzct** [comment]
Addressed @jodh-intel's comments, and delete DNM flag. ;) @bergwolf

**Pull request merged**